util/logging.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+GRAY='\033[0;90m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+function verbose() {
+  printf "\n${GRAY}[  $(date +'%Y-%m-%d %H:%M:%S')  VERB  ]  $1${NC}\n"
```

Comment:
I'd probably drop the trailing newline.

e.g. output is currently

[  2017-11-08 11:18:44  INFO  ]  Creating docker instance for dev server

[  2017-11-08 16:18:44  INFO  ]  SUCCESS: Regen from protos
Setting up watches.  Beware: since -r was given, this may take a while!
Watches established.

Reply:
Could do. I found that I sometimes didn't get a final newline before my prompt when running scripts, or logging would show up unaligned because it was output on the same line as some dangling output from a sub-process. Both are annoying.